src/components/oppsummeringAvKlage/OppsummeringAvKlage.tsx

```diff
@@ -98,6 +94,10 @@ const FormkravInfo = (props: { klage: Klage; klagensVedtak: Vedtak }) => {
                             : formatMessage('label.nei')}
                     </BodyShort>
                 </div>
+                <div>
```

Comment:
Uten å vite nøyaktig hvordan dette skal se ut rent visuselt så tenker jeg umiddelbart at alle disse `<Label>`-`<BodyShort>`-parene kanskje burde vært generalisert. Slik at dersom vi i fremtiden må endre noe, så holder det å endre ett sted. Mest sannsynlig skal det se likt ut som et annet sted i løsningen vår og. Kunne man gjenbrukt `Oppsummeringsfelt` som har samme paret feks? 😄

Reply:
Har lagt opp til at Oppsummeringsparet kan bli vist på den måten som er ønsket for klage. kan sikkert gjøres bedre, men er ihvertfall et steg i riktig retning